src/services/lnd/onchain-service.ts

```diff
@@ -62,6 +79,17 @@ export const OnChainService = (
     }
   }
 
+  const findOnChainFee = async ({
```

Comment:
shouldn't this belong to domain?

Reply:
this is from service, I encapsulated the use of internal lightning library using listOutgoingTransactions but this is an specific implementation (after on-chain migration probably will be queried directly to bitcoind)

Reply:
note: we got the input that bitcoind may not scale well in term of how many onchain address it can monitor (it's made as a individual wallet, not an entreprise that could monitor millions of addresses), so not sure we'll migrate anytime soon